test/functorch/test_vmap.py

```diff
@@ -4625,6 +4626,57 @@ def f(x):
         with self.assertRaisesRegex(RuntimeError, "autograd.Function"):
             transform(input)
 
+class TestVmapDeviceType(Namespace.TestVmapBase):
+    def _vmap_test(self, *args, **kwargs):
+        return _vmap_test(self, *args, **kwargs)
+
+    def test_check_tensor(self, device):
```

Comment:
Please use `check_vmap_fallback` to assert that the vmap fallback was not invoked. vmap fallback means we didn't add full support for these operations